core-bundle/src/Security/User/ContaoUserProvider.php

```diff
@@ -58,8 +58,6 @@ public function __construct(ContaoFramework $framework, SessionInterface $sessio
      */
     public function loadUserByUsername($username): User
     {
-        trigger_deprecation('contao/core-bundle', '4.13', 'Using "ContaoUserProvider::loadUserByUsername()" has been deprecated and will no longer work in Contao 5.0. Use "ContaoUserProvider::loadUserByIdentifier()" instead.');
```

Comment:
As long as we support Symfony 5.x we will have to implement the method. It's deprecated since 5.3 and removed in 6.0. Don't know how to proceed from here?

Reply:
May we also deprecate it in 5.x and drop it in 6.0 as we support Symfony 6.x only then?